src/Microsoft.DotNet.Build.Tasks/VersionTools/UpdateToRemoteDependencies.cs

```diff
@@ -59,7 +86,7 @@ protected GitHubClient CreateClient(bool allowAnonymous)
             }
             else
             {
-                gitHubAuth = new GitHubAuth(GitHubAuthToken, GitHubUser, GitHubEmail);
+                gitHubAuth = new GitHubAuth(GitHubAuthToken, GitHubUser, null);
```

Comment:
Why don't you want/need to pass email here?

Reply:
Email isn't used for anything `UpdateToRemoteDependencies` does, so I'm making that clear by removing that task property.

I just noticed that this argument defaults to `null`, so I'm removing the explicit `null` here now.